ggml.c

```diff
@@ -11766,15 +12036,15 @@ static void ggml_compute_forward_clamp_f32(
         const struct ggml_tensor * src1,
         struct ggml_tensor * dst) {
     assert(params->ith == 0);
-    assert(src1->type == GGML_TYPE_I32);
+    assert(src1->type == GGML_TYPE_F32);
     assert(ggml_nelements(src1) == 2);
```

Comment:
At least the last two should probably be `GGML_ASSERT` so that they are checked on release builds.

Reply:
Yea, `GGML_ASSERT` is also OK for minor equals checks. Technically `assert` is better here, but went ahead with the more common `GGML_ASSERT` based on other similar ops